src/enums.h

```diff
@@ -245,5 +245,15 @@ typedef enum {
 } htcp_clr_reason;
 #endif /* USE_HTCP */
 
+typedef enum {
+#if USE_OPENSSL
+    xaStepTlsBump1,
+    xaStepTlsBump2,
+    xaStepTlsBump3,
+#endif
+    xaStepGeneratingConnect,
+    xaStepValuesEnd
+} XactionSteps;
```

Comment:
The type stores at most one step value, not a collection of them:

```suggestion
} XactionStep;
```

Reply:
commit ccd68c4b68